src/main.js

```diff
@@ -28,4 +28,8 @@ exports.toJS = helpers.toJS
 exports.toImmutable = helpers.toImmutable
 exports.isImmutable = helpers.isImmutable
 
-exports.createReactMixin = require('./create-react-mixin')
+// @todo: improve how to load these addons
+// because react is defined as externals in webpack, not huge loss of space
+// if they aren't used
+exports.ReactMixin = require('../addons/react-mixin')
```

Comment:
I may be missing something here, but what is the need for exporting a single file that contains all of these addons.  As NuclearJS gains more addons it seems like we should make them as modular as possible since some of them may not be used with others.